client/state/jetpack-connection-health/selectors/is-jetpack-connection-problem.js

```diff
@@ -67,10 +67,13 @@ export const useIsJetpackConnectionProblem = ( siteId ) => {
 /**
  * React HOC to check if the current site has possible Jetpack connection problem.
  */
-export const withJetpackConnectionProblem = ( Wrapped ) => ( props ) => {
-	const siteId = useSelector( getSelectedSiteId );
-	const isPossibleConnectionProblem = useIsJetpackConnectionProblem( siteId );
-	return (
-		<Wrapped { ...props } isPossibleJetpackConnectionProblem={ isPossibleConnectionProblem } />
-	);
+export const withJetpackConnectionProblem = ( Wrapped ) => {
+	const WrappedWithJetpackConnectionProblem = ( props ) => {
+		const siteId = useSelector( getSelectedSiteId );
+		const isPossibleConnectionProblem = useIsJetpackConnectionProblem( siteId );
+		return (
+			<Wrapped { ...props } isPossibleJetpackConnectionProblem={ isPossibleConnectionProblem } />
+		);
+	};
+	return WrappedWithJetpackConnectionProblem;
```

Comment:
This change is not connected to the fix in teh PR, but we have a new lint rule in Calypso and since I touched this module, I got that error so I had to fix it to pass the CI checks:
<img width="1039" height="263" alt="Screenshot 2025-07-22 at 14 13 44" src="https://github.com/user-attachments/assets/84f68ca7-10b0-4bb1-8322-029cb9008315" />